app/src/main/java/de/westnordost/streetcomplete/quests/QuestsModule.kt

```diff
@@ -466,4 +467,7 @@ fun questTypeRegistry(
 
     /* at the very last because it can be difficult to ascertain during day. used by OsmAnd if "Street lighting" is enabled. (Configure map, Map rendering, Details) */
     AddWayLit(),
+
+    /* added it at the end because it could be not easy to understand this quest */
+    AddAccessPointRef(),
```

Comment:
This is not a reason to put it at the end. It should be put according to how far you have to be away from it to be able to answer this.
So, it should be somewhere near AddPostboxRef etc